src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
 public class Duke {
+    public static void printSplit(){
+        System.out.println("    ✻❊✽✼❉✱✲✾❃❋❈❆✿❀❁");
+    }
+
+
+    public static void printBye() {
+        printSplit();
+        System.out.println("    Bye. Hope to see you again soon!");
+        printSplit();
+    }
+
     public static void main(String[] args) {
+        Task task = new Task();
```

Comment:
I like how the Task class managers the list of tasks in the program. Maybe in the future it could be separated into two classes to help with encapsulation. A task manager could manage the list, and the Task class could just contain the member variables and methods that belong to a task